pkg/configtx/orderer.go

```diff
@@ -227,9 +227,9 @@ func (c *ConfigTx) OrdererConfiguration() (Orderer, error) {
 	}, nil
 }
 
-// AddOrdererOrg adds a organization to an existing config's Orderer configuration.
-// Will not error if organization already exists.
-func (c *ConfigTx) AddOrdererOrg(org Organization) error {
+// SetOrdererOrg adds a organization to an existing config's Orderer configuration.
```

Comment:
Similar to other comments for application and consortium org

Reply:
done